.github/workflows/build-web-extension.yml

```diff
@@ -0,0 +1,62 @@
+name: Build Web Extension
+
+on:
+    workflow_dispatch:
+        inputs:
+            server_url_variable:
+                type: choice
+                description: "Server URL Variable"
+                default: "LOCAL_PL_SERVER_URL"
+                required: true
+                options:
+                    - "LOCAL_PL_SERVER_URL"
+                    - "BETA_PL_SERVER_URL"
+                    - "PRODUCTION_PL_SERVER_URL"
```

Comment:
@MaKleSoft I don't have permissions to add these secrets to GitHub, please do so.